doc/installation-ce.rst

```diff
@@ -16,11 +16,16 @@ The easy way to install the dependencies is to use `pip <https://pypi.python.org
 
 On Debian, you can run ::
 
- aptitude install python-pip python-dev
+ apt update
+ apt install --no-install-recommends -y apt-utils
+ apt install --no-install-recommends -y make wget git gcc libc-dev gunicorn python-sphinx gnupg2 libsasl2-dev libldap2-dev libssl-dev python-pip python-dev
```

Comment:
Do you remember why we need libsasl2-dev. If I remember we could skip it by installing the debian packages for one of the python module.